### PR TITLE
Create 23-07-30-CurveFi

### PR DESCRIPTION
**Pull request by Adelaar5**

### CODE REVIEW

File: content/attacks/posts/23-07-30-CurveFi [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  - DeFi Exchange
+  - Liquidity Pool
+attack-types: Re-entrancy Guard Exploit
+title: "Curve’s Liquidity Pools Exploited for Over $70 Million"
+
+## Summary: 
+
+On July 30, a still-unidentified hacker drained over $60 million from stable pools that decentralized exchange Curve uses to offer exchange of tokens. 
+
+Curve ranks among the most esteemed and reliable DEXes and relies on automated market makers in much the same way as Uniswap. Though it is still functioning, Curve has seen an exodus of funds since the hack, which has in turn rattled confidence in DeFi writ large.
```
Comment (jhirschkorn) [inline content/attacks/posts/23-07-30-CurveFi:15]: Remove the second half of the last sentence. not necessary

File: content/attacks/posts/23-07-30-CurveFi [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Losses
+
+The attack drained $62 million in tokens, mostly wrapped Ether. It targeted four of Curve’s Ether-linked pools: pETH, msETH, alETH and Curve’s native token. 
+
+As illustrated below, the enigmatic exploiter has been returning some of the funds.
```
Comment (jhirschkorn) [inline content/attacks/posts/23-07-30-CurveFi:21]: remove "enigmatic"

File: content/attacks/posts/23-07-30-CurveFi [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Aug. 11: Curve [puts out](https://twitter.com/CurveFinance/status/1689995987073261568) post-hack safety report, but also [says](https://twitter.com/CurveFinance/status/1689995852411027456) 70% of funds affected have been recovered. 
+
+Aug. 18: Metronome Synth [releases](https://metronomedao.medium.com/vyper-curve-exploit-post-mortem-6a42d6d9fa07) its own post-mortem. 
+
+## Attackers: 
+
+Unknown origins; Curve Finance has offered a $1.85 million bounty to anyone who can identify the hacker. 
+
+Just under $16 million of the hacked funds are still sitting in 0xb1c33b391c2569b737ec387e731e88589e8ec148 as of 08.22.
```
Comment (jhirschkorn) [inline content/attacks/posts/23-07-30-CurveFi:55]: full date in DD MMM YYYY format

File: content/attacks/posts/23-07-30-CurveFi [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Aug. 18: Metronome Synth [releases](https://metronomedao.medium.com/vyper-curve-exploit-post-mortem-6a42d6d9fa07) its own post-mortem. 
+
+## Attackers: 
+
+Unknown origins; Curve Finance has offered a $1.85 million bounty to anyone who can identify the hacker. 
+
+Just under $16 million of the hacked funds are still sitting in 0xb1c33b391c2569b737ec387e731e88589e8ec148 as of 08.22. 
+
+The hacker did, however, return around $12.7 million to Alchemix, accompanied with a cryptic [message](https://etherscan.io/idm?tx=0x23c4799784c91023204bd68a94ec7a963486f2485dc43c13d8b804d5301b8041) about not wanting to destroy the project. The syntax of the message suggests Russian or Eastern European.
```
Comment (jhirschkorn) [inline content/attacks/posts/23-07-30-CurveFi:57]: dont speculate about the message. remove the last sentence.

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by Adelaar5** [comment]
Hi Jules, I've added all of your suggestions. Regarding the quality check, seems like it's flagging statements involving info that it just can't find, even links and wallet addresses that are listed. Do you want responses to each red x? 

Upon review, I'd say the facts are tight, barring a few unknown wallets floating around and some flexibility on the dollar amounts (depending on who's doing the conversions; really tough for example to give msETH a valuation, given it's never had much volume to speak of).

**Comment by jhirschkorn** [comment]
> Hi Jules, I've added all of your suggestions. Regarding the quality check, seems like it's flagging statements involving info that it just can't find, even links and wallet addresses that are listed. Do you want responses to each red x?
> 
> Upon review, I'd say the facts are tight, barring a few unknown wallets floating around and some flexibility on the dollar amounts (depending on who's doing the conversions; really tough for example to give msETH a valuation, given it's never had much volume to speak of).

the bot is pretty dumb so dont spend too much time fighting it. just give the flagged statements a quick check.  Ignore it if it doesnt check out or add an additional citation if it might make things clearer. you already have a number of good citations in the article tho.  i will review again later today and merge. thanks for your submission.

File: content/attacks/posts/23-07-30-CurveFi [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+Aug. 11: Curve [puts out](https://twitter.com/CurveFinance/status/1689995987073261568) post-hack safety report, but also [says](https://twitter.com/CurveFinance/status/1689995852411027456) 70% of funds affected have been recovered. 
+
+Aug. 18: Metronome Synth [releases](https://metronomedao.medium.com/vyper-curve-exploit-post-mortem-6a42d6d9fa07) its own post-mortem. 
+
+## Attackers: 
+
+Unknown origins; Curve Finance has offered a $1.85 million bounty to anyone who can identify the hacker. 
+
+Just under $16 million of the hacked funds are still sitting in 0xb1c33b391c2569b737ec387e731e88589e8ec148 as of 25 08 2023.
```
Comment (jhirschkorn) [inline content/attacks/posts/23-07-30-CurveFi:55]: date format should read 25 AUG 2023

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by jhirschkorn** [comment]
one more small fix

File: content/attacks/posts/23-07-30-CurveFi (hunk 5 above)
Comment (Adelaar5) [reply content/attacks/posts/23-07-30-CurveFi]: Ah, misunderstood that. Have changed; all caps for the month right?

**Review by jhirschkorn — APPROVED** [review]

**Comment by albina-at-inca** [comment]
> ## How to Claim Bounty
> Email your bitcoin/lightning payment address to [wiki.challenge.bounties@dn.institute](mailto:wiki.challenge.bounties@dn.institute). Expect all payouts to be completed by the end of the month.

@Adelaar5 can you send me your BTC address? Please include the QR code.

**Comment by Adelaar5** [comment]
@albina-at-inca Just emailed. Please let me know if there are any issues!

**Comment by albina-at-inca** [comment]
> @albina-at-inca Just emailed. Please let me know if there are any issues!

Received, thank you.

**Pull request merged**